comm_norms.adoc

```diff
@@ -12,18 +12,30 @@ And we will conclude with some observations on the ways ever-changing tooling an
 As the number of free and open source software projects has increased over time, the communication tools projects use—both to connect with their users and to coordinate their developers and contributors—have evolved.
 Internet Relay Chat (IRC) was once the default choice for real-time project chat.
 Many large projects conducted development discussions solely via their electronic mailing lists, https://en.wikipedia.org/wiki/Freecode[Freshmeat] was the place to hear about new project releases or updates to existing projects, and discussions about feature design and project improvements occurred via email and were included in a project's mailing list archives.
+PD--The sentence in line 14 is quite long. Consider breaking into two sentences (most likely between "projects" and "discussion").
 
 Today, the landscape has shifted:
+PD--The bullets that follow are quite long. Consider either paring them down so they are the length of one to two relatively simple sentences or reformat into paragraphs.
```

Comment:
I am open to reformatting into paragraphs for a later revision of the text, but I am OK with them going live as they currently are to make the 1.0 release.

Reply:
Easy enough to drop the bullets this time. I'm going to add a few words to the lead-in paragraph to make it lead in better without it being bullets.